oioioi/problems/management/commands/create_mock_competition.py

```diff
@@ -0,0 +1,204 @@
+
+# Helper command, which finds the the smallest natural number n, such that
+# no object from the model model_class has a field equal to [base_value]n.
+# This function is to be used for finding anyname that isn't
+# used in the database yet.
+def get_unique_field_value(model_class, field_name, base_value):
+    regex_pattern = base_value + r"(\d+)"
+
+    filter_kwargs = {f"{field_name}__regex": f"^{regex_pattern}"}
```

Comment:
That's an overkill, startswith should be enough: https://docs.djangoproject.com/en/5.1/ref/models/querysets/#startswith